gui/default/syncthing/core/syncthingController.js

```diff
@@ -1679,7 +1683,9 @@ angular.module('syncthing.core')
                         $scope.currentDevice = p.data;
                         $scope.currentDevice.name = name;
                         $scope.currentDevice.deviceID = deviceID;
-                        $scope.currentDevice._editing = "add";
+                        if (!deviceID) {
+                            $scope.currentDevice._editing = "new";
+                        };
```

Comment:
You're not setting the variable at all now, while previously you had the special `add-pending` value.  That's error prone, please go back to setting it explicitly.

I don't see the need to rename the value from "add" to "new".  It doesn't really matter what the value is, but it's usually good to keep the changes minimal.  Especially if you want to get the review process done in a reasonable time frame :wink:

Reply:
The variable is now set. I've kept the `new` name though. The reason behind it is that the helper functions can be named `editingDeviceNew` and `editingDevicePending`, which seem, at least for me, easier to understand than something like `editingDeviceAdd` and `editingDeviceAddPending` or similar.

What do you think?